packages/api/src/schema/schema.yaml

```
@@ -1151,6 +1170,22 @@ components:
                 assetSpec:
                   type: object
                   $ref: "#/components/schemas/asset"
+            export:
+              type: object
+              additionalProperties: true
+              description: Output of the export task
+              properties:
+                videoFilePath:
+                  type: string
+                  description: could be a url or a IPFS CID
+                metadataFilePath:
+                  type: string
+                  description: could be a url or a IPFS CID
+                assetSpec:
+                  type: object
+                  $ref: "#/components/schemas/asset"
+                ipfs:
+                  type: boolean
```

I'd go with
```suggestion
                ipfs:
                  type: object
                  properties:
                    videoFileCid:
                      type: string
                      description: IPFS CID of the exported video file
                    erc1155MetadataCid:
                      type: string
                      description: IPFS CID of the default metadata exported for the video
```

custom export has no output I guess. Can you think of any?

---

Oh you already defined the IPFS  object for the output! Cool, looks great. Nothing comes to mind right not for the custome export output